src/Serilog/Project.json

```diff
@@ -8,7 +8,6 @@
     "frameworks": {
         "net40": {
             "compilationOptions": {
-                "define": [ "NET40" ],
```

Comment:
These are added automatically by DNX, i.e. for `net45`, a `NET45` preprocessor directive is added.